simple_centralized_greedy.py

```diff
@@ -0,0 +1,69 @@
+# This file implement simplified greedy replication algorithm.
+
+# Python import
+import random
+import os
+import sys
+import time
+
+# Project imports
+sys.path.insert(0, os.path.join(os.path.dirname(os.path.realpath(__file__)), 'aggregator'))
+import util
+from aggregator import Aggregator
+
+class SimpleCentralizedGreedy:
+
+  def __init__(self):
+    self.aggregator = Aggregator() # to retrive server logs
+    self.server_set = set(util.retrieve_server_list()) # [server_ip, ]
+    self.content_set = set([])
```

Comment:
Oh you don't have to do that actually. You can do something like `set_a = set()`, I think.